steering_controllers/steering_odometry/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.8)
+project(steering_odometry)
```

Comment:
There is no need for a separate package. This can be added directly into `steering_controllers` package and potentially could be compiled into an interal library.